specifications/cdsc-wg1-01.md

```diff
@@ -1 +1,292 @@
+* `coverage` - _URL_ - (OPTIONAL) Where to find the [Coverage Endpoint](#coverage-endpoint)
+* `related_metadata` - _Array[URL]_ - (OPTIONAL) Where to find other related [Metadata Endpoints](#metadata-endpoint)
+
+Values and arrays of values in the metadata object is assumed to only related to the array of capabilities the Server is offering. For example, if a utility that offers both electricity and natural gas, but is only offering data access capabilities to electric service accounts, the Server only needs to include `electricity` in its `commodity_types` array and not `natural_gas`.
+
+### 3.3. Metadata Endpoint Organization <a id="metadata-endpoint-organization" href="#metadata-endpoint-organization" class="permalink">🔗</a>
+
+Many utilities and other similar entities are structured as multiple divisions, operating companies, territories, legal constructs, or other types of groupings. These organizational complexities and legal structures can make it difficult to publish a single representative metadata endpoint for an entire entity.
+
+When utilities and other similar entities wish to publicly provide multiple metadata objects that more cleanly define the metadata and capabilities for their entity, their Server MUST publish a single metadata endpoint at the [well-known URI](metadata-well-known-uri) and within that metadata object include an array of `related_metadata` URLs that link the Client to the various subdivided or related metadata endpoints.
+
+It is acceptable for the initial well-known metadata endpoint object to include a minimal or empty array of `capabilities`, as it represents the entire entity, and leave enumerating the `capabilities` to the related metadata endpoints.
+
+### 3.4. Infrastructure Types <a id="infrastructure-types" href="#infrastructure-types" class="permalink">🔗</a>
+
+Infrastructure types are general categories of utility and energy grid entities that are likely to need to provide access to metadata and capabilities under this specification. The following list of strings are an enumerated set of infrastructure types that MAY be included in the `infrastructure_types` array of values in the metadata object.
+
+* `distribution_utility` - A electric, natural gas, and/or water utility that distributes the utility service to the end customer (e.g. "who maintains the wires to your house")
+* `metering_provider` - An entity that maintains the utility metering infrastructure and/or reads end customer meters (e.g. "who reads your meter")
+* `supplier` - An energy supplier for the end customer (e.g. "who you buy power from")
+* `central_repository` - A government mandated entity that collects customer usage information into a centralized repository for re-distribution (e.g. a state-level meter data warehouse for deregulated jurisdictions)
+
```

Comment:
From @halliecramer1 ([on April 14, 2023](https://github.com/carbon-data-specification/Customer-Data/pull/23#discussion_r1167327730)):

---

Add `energy_solutions_provider`, e.g. a Scheduler in the U.S. or a Balancing Responsible Party in Europe, who can provide data on specific transmission level generation assets

---

Reply:
@halliecramer1 Thanks for the suggestion! Updated `infrastructure_type` to include `service_provider`. https://daniel-utilityapi.github.io/Connectivity/specs/cdsc-wg1-01/#infrastructure-types